sway/input/input-manager.c

```diff
@@ -541,6 +541,33 @@ static void handle_new_input(struct wl_listener *listener, void *data) {
 	sway_log(SWAY_DEBUG, "adding device: '%s'",
 		input_device->identifier);
 
+	const char *device_type = input_device_get_type(input_device);
+	struct input_config *type_config = NULL;
+	for (int i = 0; i < config->input_type_configs->length; i++) {
+		struct input_config *ic = config->input_type_configs->items[i];
+		if (strcmp(ic->identifier + 5, device_type) == 0) {
+			type_config = ic;
+			break;
+		}
+	}
+
+	// Merge new device config on top of type wildcard, if the prior exists
```

Comment:
Sorry missed this one before. This block needs to be wrapped in a `if (type_config) {` otherwise sway segfaults when trying to merge the NULL type config on top of `current` (line 559)